src/codegen/intermediate_representation.c

```diff
@@ -643,13 +638,15 @@ IRInstruction *ir_create_static
   v->name = string_dup(name);
   v->type = type;
   v->cached_size = type_sizeof(type);
-  v->cached_alignment = 8; /// TODO.
+  // TODO: Don't just use 8 byte alignment for every static.
+  // Should we just use the nearest larger (or equal) power of two, in the generic case?
+  v->cached_alignment = 8;
```

Comment:
There should probably be a `type_alignof` function in `ast.h`/`ast.c` that we can use to query the alignment of a type. When emitting static vars, the *backend* has to make sure that all of them are properly aligned. E.g. if we emit a `byte` and then an `integer`, we have to tell the assembler to align the `integer` to 8 bytes. And actually, we might have to do that in any case anyway, because I don’t think we can rely on the first static to already be aligned properly...

Reply:
Also, on that note, the idea behind `cached_size` and `cached_alignment` was that sema would compute the size/alignment and then store it in there so we can easily access it later w/o having to recompute it every time. Not sure if that is that good of an idea tho honestly, simply because that information might become outdated if something about the type changes, and I feel like debugging something like that would be rather horrible. 

We can always add it back if, for some reason, `type_sizeof()` or `type_alignof()` end up being a huge bottleneck after all, but rn, this just screams premature optimisation now that I think about it...

Reply:
Also added `type_alignof` with just a stubbed out `TODO` implementation in c9ccbb21fb41ed25d4b739eeeeba5da82b39093a